authors/apps/authentication/serializers.py

```diff
@@ -17,8 +17,8 @@ class RegistrationSerializer(serializers.ModelSerializer):
         write_only=True,
         required = True,
         validators =[RegexValidator(
-            regex='^[a-zA-Z0-9]*',
-            message = 'Password should be alphanumeric',
+            regex="^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$",
+            message = "Please ensure your password contains at least one letter and one numeral",
```

Comment:
Please ensure you lint your code using PEP8. I am getting 15 warnings from this file

Reply:
Also, there are some unused comments on lines 209-211